From 18fe48247591440c6a49d41c573cd011b541cc83 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:43 -0400
Subject: [usb] Revert: quirks: add touchscreen that is dazzeled by remote wakeup

Message-id: <1410887463-20674-212-git-send-email-dzickus@redhat.com>
Patchwork-id: 95216
O-Subject: [RHEL7 PATCH 211/591] Revert "USB: quirks: add touchscreen that is dazzeled by remote wakeup"
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit e0e6a356a0b41375b729c511caa99d3ec019640e
Author: Oliver Neukum <oneukum@suse.de>
Date:   Fri Dec 6 16:09:27 2013 +0100

    Revert "USB: quirks: add touchscreen that is dazzeled by remote wakeup"

    This reverts commit 614ced91fc6fbb5a1cdd12f0f1b6c9197d9f1350.
    The units on this was seen were prototypes and the issue is
    not seen on younger units.

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/quirks.c b/drivers/usb/core/quirks.c
index 12924db..8f37063 100644
--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -98,9 +98,6 @@ static const struct usb_device_id usb_quirk_list[] = {
  /* Alcor Micro Corp. Hub */
  { USB_DEVICE(0x058f, 0x9254), .driver_info = USB_QUIRK_RESET_RESUME },
 
- /* MicroTouch Systems touchscreen */
- { USB_DEVICE(0x0596, 0x051e), .driver_info = USB_QUIRK_RESET_RESUME },
-
  /* appletouch */
  { USB_DEVICE(0x05ac, 0x021a), .driver_info = USB_QUIRK_RESET_RESUME },
 
-- 
1.7.1